rst/indigo/release-notes/indigo-1.8.0.md

```diff
@@ -0,0 +1,35 @@
+# Indigo 1.8.0
+
+*17 Nov. 2022*
+
+## Features
+* Core: Added indigo option "smiles-saving-format" to save daylight smiles (#876)
+* Core: Support for generic and polymer s-groups for extended SMILES (#874)
```

Comment:
Need to convert #874 into [#874](https://github.com/epam/Indigo/issues/874)
there is auto-generation for # notation

Reply:
fixed